Libs/vtkSegmentationCore/vtkSegmentationConverter.cxx

```diff
@@ -303,7 +303,7 @@ std::string vtkSegmentationConverter::GetConversionParameterDescription(const st
 //----------------------------------------------------------------------------
 vtkSegmentationConverter::ConversionPathType vtkSegmentationConverter::GetCheapestPath(const ConversionPathAndCostListType &pathsCosts)
 {
-  unsigned int cheapestPathCost = vtkSegmentationConverterRule::GetConversionInfiniteCost();
```

Comment:
Would it make more sense to change vtkSegmentationConverterRule::GetConversionInfiniteCost()? If the value it returns causes bugs, then it should not be left as is.

Reply:
I don't think the value of GetConversionInfiniteCost is an issue. 

It was only a problem for initializing the cheapestPathCost because it was also used as the cost of planar contour to ribbon conversion. It wouldn't matter what the value was in this case, it would still cause the issue.

Reply:
I see. Thanks for the explanation